src/Tribe/Cost_Utils.php

```diff
@@ -128,9 +128,11 @@ public function maybe_format_with_currency( $cost, $event = null, $currency_symb
 		// be sure to account for european formats in decimals, and thousands separators
 		if ( is_numeric( str_replace( $this->get_separators(), '', $cost ) ) ) {
 			$reverse_position = null;
-			if ( null !== $currency_position ) {
+			// Sometimes currency_position gets passed as an empty string, hence the checking for empty() as well here.
+			if ( null !== $currency_position && ! empty( $currency_position ) ) {
```

Comment:
Could this be simplified to `( ! empty( $currency_position ) )` (ie, ditch the explicit null check)?